mezzanine/core/templatetags/mezzanine_tags.py

```diff
@@ -109,8 +109,7 @@ def errors_for(context, form):
     """
     Renders an alert if the form has any errors.
     """
-    context["form"] = form
-    return context
+    return {"form": form}
 
```

Comment:
Yes, but that is how inclusion tags are supposed to work - https://docs.djangoproject.com/en/1.9/howto/custom-template-tags/#inclusion-tags - you are supposed to return a simple dictionary containing only the data needed, not the whole context.

The only value used by the `form_errors.html` template is `form`, so behaviour is preserved.

The alternatives:
1) what happened before - that you actually mutate the context that is passed in, which is a bad idea for the sake of the calling context, because it potentially has to deal with the side effects of the inclusion tag, and the template gets a nested Context object.
2) copy all the data from the passed in Context obejct into a new Context object. This seems like a bad idea when you only need one value from it.